sdk/spring/integration-matrix.json

```diff
@@ -6,7 +6,7 @@
         "ProfileFlag": "-P integration-test-azure",
         "Pool": "azsdk-pool-mms-ubuntu-2004-general",
         "OSVmImage": "MMSUbuntu2004",
-        "JavaTestVersion": "1.11"
+        "JavaTestVersion": "1.17"
```

Comment:
I guess adding Java 17 test coverage is goodness - but removing Java 11 test coverage is bad. Vast majority of our customers still uses Java 8 or 11

Reply:
+1 to Fabian. Also, Java 17 doesn't work with spring-data-cosmos because of UUID support as Id type. There is an open bug https://github.com/Azure/azure-sdk-for-java/issues/28926